`make distcheck` fails

`make distcheck` on at least x86_64-linux is failing with the following message.
```
../../src/coredump/_UCD_internal.h:49:28: fatal error: ucd_file_table.h: No such file or directory
```

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -71,7 +71,9 @@ libunwind_coredump_la_SOURCES = \
 libunwind_coredump_la_LDFLAGS = $(COMMON_SO_LDFLAGS) \
 				-version-info $(COREDUMP_SO_VERSION)
 libunwind_coredump_la_LIBADD = $(LIBLZMA) $(LIBZ)
-noinst_HEADERS += coredump/_UCD_internal.h coredump/_UCD_lib.h
+noinst_HEADERS += coredump/_UCD_internal.h \
+                  coredump/_UCD_lib.h \
+                  coredump/ucd_file_table.h
 
 ### libunwind-setjmp:
 libunwind_setjmp_la_LDFLAGS		= $(COMMON_SO_LDFLAGS)		     \

